src/account/Support.tsx

```diff
@@ -20,8 +22,15 @@ const Support = () => {
   const { t } = useTranslation()
 
   return (
-    <SafeAreaView style={styles.container}>
-      <DrawerTopBar />
+    <SafeAreaView
+      style={styles.container}
+      edges={
+        getFeatureGate(StatsigFeatureGates.USE_TAB_NAVIGATOR)
```

Comment:
Instead of getting the gate twice we can assign it to a constant. 
`const isTabNav = getFeatureGate(StatsigFeatureGates.USE_TAB_NAVIGATOR)`